compiler/noirc_evaluator/src/ssa/ssa_gen/mod.rs

```diff
@@ -587,6 +588,38 @@ impl<'a> FunctionContext<'a> {
         Ok(Self::unit_value())
     }
 
+    /// Codegens a loop, creating three new blocks in the process.
+    /// The return value of a loop is always a unit literal.
+    ///
+    /// For example, the loop `loop { body }` is codegen'd as:
+    ///
+    /// ```text
+    ///   br loop_body()
+    /// loop_body():
+    ///   v3 = ... codegen body ...
+    ///   br loop_body(v4)
```

Comment:
```suggestion
    ///   br loop_body(v3)
```

Reply:
Good catch! I changed it to not pass any value.